Cargo.toml

```diff
@@ -89,7 +89,7 @@ base64 = "0.22"
 bencher = "0.1.5"
 bzip2 = "0.4.4"
 once_cell = "1.17.2"
-rcgen = { version = "0.12.0" }
+rcgen = { version = "0.13", features = ["aws_lc_rs"] }
```

Comment:
Should this have `default-features = false` to avoid a `ring` dependency?

Reply:
Fixed.